src/team.cpp

```diff
@@ -976,6 +977,18 @@ std::string team::get_side_color_id(unsigned side)
 	}
 }
 
+const t_string team::get_side_color_name_for_UI(unsigned side)
+{
+	const std::string& color_id = team::get_side_color_id(side);
+	const auto& rgb_name = game_config::team_rgb_name[color_id];
+	if(rgb_name.empty())
+		// TRANSLATORS: $color_id is the internal identifier of a side color, for example, 'lightred'.
+		// Translate the quotation marks only.
+		return VGETTEXT("\"$color_id\"", {{ "color_id", color_id }});
```

Comment:
Sorry for not spotting this until it merged, but if the quotation marks should be translated then this should use typographical quotation marks in English.

Please give the translators a hint about how to test, as in how to reach this with rgb_name empty.

Reply:
> Sorry for not spotting this until it merged, but if the quotation marks should be translated then this should use typographical quotation marks in English.

No problem, done.

> Please give the translators a hint about how to test, as in how to reach this with rgb_name empty.

Translators can test this by deleting the `name=_ "Red"` line from data/core/team-colors.cfg, running `wesnoth --multiplayer`, and opening the tooltip of the red side's top bar or sidebar flag icon. Shall I add this information to the po comment? Wouldn't it be better to say in the po comment how the string is used, to let translators do their work without having to edit game files and start wesnoth in the middle of their translating?

Reply:
@Wedge009 and @ivanovic , please would you comment on what would be best for the translators? I assume that they test in-game.

Reply:
I'm not sure many translators would also test in-game - this would mean they have the development skills to compile the translations themselves and add it to the game.

As for making it clear on what to translate, we had the discussion not too long ago relating to translation of commands where the command itself was to be left untranslated. Although here it's a bit more tricky since we're using color_id in both places. Actually, I'm not even sure what's supposed to be translated here - is it the $color_id part?

Reply:
> Actually, I'm not even sure what's supposed to be translated here - is it the $color_id part?

If I'm understanding correctly:

English: "“$color_id”"
German: "»$color_id«"

Reply:
Exactly.

Reply:
Ohhh, translation of the quotation marks...

Reply:
Do you think someone would translate it to `"»$Farbe_id«"` (German for "color id")? I tried to prevent that with the TRANSLATORS comment, but maybe it's not clear enough.

Reply:
As long as the translator knows to read the comment for things that are obviously not normal English text - I think the comments get included in the po file and shown explicitly in something like poedit, but to be honest I don't look at them too often (but then again, maybe that's because I'm just accounting for locale differences and not doing a full cross-language translation).

Reply:
Thanks, it looks good to me now (having the typographic quotes makes it a lot clearer about what should be translated).

Reply:
Thanks for the review. Further clarified in 47b7e2a85bc.